### ISSUE TITLE
No download on selecting LinuxDeb, Daily, master (option 3)

### ISSUE THREAD

**Issue by capri99** [reisi007/Java-SI-GUI#2]
Today I've been testing the Java-SI-GUI (v3) and these are the results:

```
$ java -jar siguitest3.jar 
Hi! I am SI-GUI!
Choose your OS:
1 -> LinuxDeb
2 -> LinuxRPM
1
You chose LinuxDeb - x86_64!
Fetching possible downloads in the background!
In which of the following dowloadTypes are you interested
1 -> Archive
2 -> Testing
3 -> Stable
4 -> Daily
4
Which files do you want to download. Type 'm' for the main program, 's' for the SDK and 'h' for the help
m
You have shown interest in Daily.
We now wait for the fetching to complete.Download finished!
1 -> libreoffice-4-4Linux-rpm_deb-x86_64@46-TDF Linux/x86_64 @ http://dev-builds.libreoffice.org/daily/libreoffice-4-4/Linux-rpm_deb-x86_64@46-TDF/current/
2 -> libreoffice-5-0Linux-rpm_deb-x86_64@46-TDF Linux/x86_64 @ http://dev-builds.libreoffice.org/daily/libreoffice-5-0/Linux-rpm_deb-x86_64@46-TDF/current/
3 -> masterLinux-rpm_deb-x86_64@46-TDF Linux/x86_64 @ http://dev-builds.libreoffice.org/daily/master/Linux-rpm_deb-x86_64@46-TDF/current/
4 -> masterLinux-rpm_deb-x86_64@46-TDF-dbg Linux/x86_64 @ http://dev-builds.libreoffice.org/daily/master/Linux-rpm_deb-x86_64@46-TDF-dbg/current/
3
Download will be started to /tmp/java-si-gui7461760813824413446
Downloading... This also hapens in the background
Download completed, program will exit now!
```

More info:

```
$ java -version
java version "1.8.0_45"
Java(TM) SE Runtime Environment (build 1.8.0_45-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.45-b02, mixed mode)
```

Also is not working if I choose option 1, 2 or 4.

Under Java 1.8.0_45, 32bits, worked ok.

**Comment by reisi007** [comment]
Could you please test downloading with the latest release (wait some minites, I will push a new release soon)

**Comment by capri99** [comment]
Ok, remember that I'm working on Linux. I've downloaded snapshot2a.jar and it appears to work but no download starts. Just pressed "Start download of selected versions" and it gets "grey".
On /tmp/ all directories created are empty.

**Comment by reisi007** [comment]
Fixed (or tried to) with 0bde2a35ebc15192e1e430bf18c48b3eec0d358f
